sleap/gui/widgets/monitor.py

```
@@ -1114,12 +1090,14 @@ def _unbind(self):
         if self.sub is not None:
             self.sub.unbind(self.sub.LAST_ENDPOINT)
             self.sub.close()
+            time.sleep(0.1)
```

Ruff (0.8.2)

Undefined name `time`

(F821)